### ISSUE TITLE
Croppie result to send to web server as a file

### ISSUE THREAD

**Issue by purukc** [Foliotek/Croppie#728]
## Expected Behavior

We are using croppie.js for image upload on one of our project. I'm  able to get my cropped result and send it to my web server as base64 string. But, I need to send the result to server as a file. I am doing this on C#.

So, Is there anyway that Croppie is able to output the result as an actual image file rather than just an encoded base64 version of it. I'm hoping someone might have an easy solution.
## Steps to Reproduce the Problem

  1.
  1.
  1.
  
## Example Link
Please recreate your issue using [JSbin](https://www.jsbin.com), [JSFiddle](https://jsfiddle.net), or [Codepen](https://codepen.io).
* Link:

## Specifications

  - Browser:
  - Version:

**Comment by bret-miller** [comment]
I don't think you can. I'm using PHP for my conversion from base64, but searching for how to do it in C# yielded this:

```
string filePath = "MyImage.jpg";
File.WriteAllBytes(filePath, Convert.FromBase64String(base64imageString));
```

In looking at my own PHP code, it seems there was something prepended to the base64 data, because I have this in my code before converting from base64:

`$imageData = substr($imageData,strpos($imageData, ",") + 1);`

I'm sure that's not hard to replicate in C# either.

**Comment by ericgoe** [comment]
This could answer your question. https://stackoverflow.com/a/1588323/7905415
